frontend/assets/javascripts/src/modules/form/subscriber.js

```diff
@@ -0,0 +1,104 @@
+define(['bean', 'ajax', 'src/modules/form/validation/display'], function (bean, ajax, display) {
+    'use strict';
+
+    var PAYMENT_OPTIONS_CONTAINER_ELEM = document.querySelector('.js-payment-options-container');
+    var SUBSCRIBER_PAYMENT_OPTIONS_CONTAINER_ELEM = document.querySelector('.js-subscriber-payment-options-container');
+    var SUBSCRIBER_ID_INPUT_ELEM = document.querySelector('.js-subscriber-id-input');
+    var SUBSCRIBER_ID_SUBMIT_ELEM = document.querySelector('.js-subscriber-id-submit');
+    var POSTCODE_ELEM = document.querySelector('.js-postcode');
+    var LAST_NAME_ELEM = document.querySelector('.js-name-last');
+    var SUBMIT_INPUT_ELEM = document.querySelector('.js-submit-input');
+    var HIDDEN_SUBSCRIBER_INPUT_ELEM = document.querySelector('.js-hidden-subscriber-input');
+
+    function init() {
+        if (SUBSCRIBER_ID_INPUT_ELEM && SUBSCRIBER_ID_SUBMIT_ELEM) {
+            bean.on(SUBSCRIBER_ID_SUBMIT_ELEM, 'click', function(event) {
+
+                event.preventDefault();
+
+                var subscriberId = SUBSCRIBER_ID_INPUT_ELEM.value,
+                    postcode = POSTCODE_ELEM.value,
+                    lastName = LAST_NAME_ELEM.value;
+
+                ajax({
```

Comment:
It's worth putting a check around this to make sure `subscriberId` isn't empty. No point making an XHR request that we no is going to fail. e.g.,

```
if(subscriberId !== '') {
    ajax…
} else {
    display.toggleErrorState({
        isValid: false,
        elem: SUBSCRIBER_ID_INPUT_ELEM,
        msg: 'Please provide your subscriber ID'
    });
}